<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>531</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1084</width>
    <height>531</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1084</width>
    <height>531</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Montserrat</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Pensamiento en voz alta</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1084</width>
     <height>531</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1084</width>
     <height>531</height>
    </size>
   </property>
   <widget class="QLabel" name="Titulo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>1741</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1741</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Integración del protocolo de pensamiento en voz alta de conducción naturalista, 
para la creación de reglas de conducción de vehículos autónomos.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_warning">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>390</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(252, 1, 7);</string>
    </property>
    <property name="text">
     <string> ¡¡¡ FORMATO ERRÓNEO !!!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLocal">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>330</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccionar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSiguiente1">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>330</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Siguiente &gt;&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ruta">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>721</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Montserrat'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; color:#7c7c7c;&quot;&gt;Ruta del archivo mp4.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Titulo_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>741</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Esta aplicación le permitirá hacer las siguientes tareas:

- Trascribir el audio de un vídeo con solo pulsar un botón. 
- Corregir las erratas de dicho texto.
- Agregar información que pueda faltar.
- Observar de forma simultánea el vídeo y el texto extraído.
- Obtener códigos de acción de conducción.
- Introducir acciones mediante patrones previamente detectados.



A continuación, debe seleccionar la nueva grabación de su ruta, debe estar en formato mp4. </string>
    </property>
   </widget>
   <widget class="QLabel" name="footer_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>500</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: blue</string>
    </property>
    <property name="text">
     <string>Trabajo de Fin de Grado de Sofía Martínez Parada</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
